conf/jasminegraph-server.properties

```diff
@@ -86,7 +86,7 @@ org.jasminegraph.scheduler.performancecollector.timing=120
 org.jasminegraph.centralstore.location=/home/tmp/centralstore
 org.jasminegraph.db.location=./metadb/jasminegraph_meta.db
 org.jasminegraph.performance.db.location=./performancedb/jasminegraph_performance.db
-
+org.jasminegraph.streaming.db.location=./streamingdb/jasminegraph_streaming.db
```

Comment:
Lets move this streaming DB to where we keep JasmineGraph's data.

Reply:
Done

Reply:
Ok